src/components/ScriptNav.jsx

```diff
@@ -1,39 +1,35 @@
-import React, { useEffect, useState } from 'react';
+import React, { useState } from 'react';
+import { deleteScript } from './api';
 
 const ScriptNav = ({ currentScript, setCurrentScript, scripts, setScripts }) => {
-  const [scriptError, setScriptError] = useState('')
+  const [scriptError, setScriptError] = useState('');
 
-  async function deleteScript(event){
+  async function deleteScriptHandler(event) {
     const deleteId = event.target.value;
-
-    try {
-      const response = await fetch('/api/script/' + deleteId, { method: 'DELETE' });
-
-      if (response.ok) {
-        setScriptError('')
-        setScripts(scripts.filter((t) => t.id.toString() !== deleteId))
-        if (deleteId === currentScript){
-          setCurrentScript(null)
-        }
-      } else {
-        setScriptError('Error when deleting the script')
+    // Delete the script from the database and remove it from the script list
+    const result = await deleteScript(deleteId);
```

Comment:
nit: may want to call this scriptId instead of deleteId to match conventions elsewhere.

Reply:
Good point, fixed!